dae-alright/src/components/Navigation/Navigation2.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
create a components folder inside this folder and move this file to that folder, then rename it to `NavAuthState`, then import it in the `Navigation` component and do
```
function Navigation(){
  return(
   ...
   if isAuthenticaticated ? <Button  text='LOGIN/SIGN UP'/>: <NavAuthState />
  )
}
```

PS: the code above it's just a `pseudo` code, so the implementation depends on how those components are built.

Reply:
Should this replace the NavisLoggedIn I created earlier because this seems to have same function

Reply:
And This navigation2 is the one for 
![image](https://user-images.githubusercontent.com/23562596/68340727-47301400-00e7-11ea-9128-59ed6365a317.png)

Reply:
YEs